### ISSUE TITLE
opam package doesn't properly uninstall itself

### ISSUE THREAD

**Issue by yminsky** [djs55/ocaml-c3#12]
which means that upgrades of the c3 package fail.

**Comment by djs55** [comment]
I'm hoping this was fixed by 824e37b39e99534d1e65c5172665fae3adac8a8d -- let me know if it's still going wrong for you.

**Comment by djs55** [comment]
I don't think this is fixed: https://travis-ci.org/mirage/mirage-www/jobs/68726184

**Comment by samoht** [comment]
I think this is fixed by using dune (see #21)
